extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/music/GoogleMusicExporter.java

```diff
@@ -256,14 +268,15 @@ private List<MusicGroup> createMusicGroups(GoogleArtist[] artists) {
   }
 
   private MusicPlaylistItem convertPlaylistItem(
-      String playlistId, GooglePlaylistItem googlePlaylistItem) {
+      String playlistId, GooglePlaylistItem googlePlaylistItem)
+      throws InvalidProtocolBufferException, ParseException {
     GoogleTrack track = googlePlaylistItem.getTrack();
     GoogleRelease release = track.getRelease();
     return new MusicPlaylistItem(
         new MusicRecording(
             track.getIsrc(),
             track.getTitle(),
-            track.getDurationMillis(),
+            track.convertDurationToMillions(),
```

Comment:
nit: up to you, but maybe MusicRecording() should do this conversion internally, since it has a model it knows it can do that to?

Reply:
Done in latest commit 7b70e28051bdc5a540f2c97e16d53578cb5601fd.